packages/synapse-react-client/src/components/CreateTableViewWizard/TableTypeSelection.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import WizardChoiceButtonGroup from '../WizardChoiceButton/WizardChoiceButtonGroup'
+import WizardChoiceButton from '../WizardChoiceButton/WizardChoiceButton'
+import WizardChoiceButtonDescription from '../WizardChoiceButton/WizardChoiceButtonDescription'
+import { Link } from '@mui/material'
+
+const TABLE_DESCRIPTION = (
+  <>
+    <WizardChoiceButtonDescription my={1.25}>
```

Comment:
As far as I can tell, `my={1.25}` is applied to each `WizardChoiceButtonDescription` instance in TableTypeSelection and ViewTypeSelection. How did you decide what to include in the default styling for `WizardChoiceButtonDescription` vs styling to apply directly to each instance of the component? Mostly just curious to understand how you're thinking about designing new components!

Reply:
I think I was trying to be flexible because I wasn't sure how it was going to look +/- docs links. But since I ended up not needing that flexibility, it would make more sense to make this the default margin. Nice catch!